src/game/character-init/characterBlueprint.js

```diff
@@ -8,18 +8,19 @@ class characterBlueprint {
         this.level = 1,
         this.experience = 0
 
+        // the methods below will become the primary handlers of stats changes
+        // battle moves will not directly access the other character's stats
+        // they will call one of the methods below
+
         this.receiveDamage = (damage) => {
```

Comment:
In de eerdere js file gebruikte je "function handleInput () {}" syntax en hier is het fat arrow, misschien ook een vd twee kiezen? ik zou sws tegen "function handleInput()" zijn omdat die achter de schermen een var creeert en je daarmee je scope vervuilt, beter is bv "const handleInput = function() {}" of meteen voor fat arrow gaan en dat in een const stoppen.

Reply:
Stom eigk zou deze comment in die andere file moeten staan maar zit ook gwn op werk nu en moet dit ff snel doorduwen xD

Reply:
goed punt van die scope vervuiling